AdyenCard/Components/Card/CardViewControllerItemsProvider.swift

```diff
@@ -27,7 +27,9 @@ extension CardViewController {
 
         private let defaultCountryCode: String
         
-        private let addressViewModelBuilder: AddressViewModelBuilder
+        internal let addressViewModelBuilder: AddressViewModelBuilder
+
+        internal var addressViewModel: AddressViewModel?
```

Comment:
I think this has to be a computed property like this:

```Swift
internal var addressViewModel: AddressViewModel { billingAddressItem.addressViewModel } 
```
because the viewModel gets updated, right?


or event better, access the `billingAddressItem` directly from the `CardViewController`